new/cookbook-abandonment.md

```diff
@@ -0,0 +1,50 @@
+---
+
+# Cookbook Abandonment On Supermarket
+
+The Chef community needs a process around handling when cookbooks on Supermarket are abandoned.  This occurs when a cookbook owner does not respond to requests to adopt the cookbook, and the cookbook has not been updated in some time.
+
+## Motivation
+
+```
+    As a supermarket user,
+    When I find a cookbook,
+    And the cookbook is up for adoption,
+    And I want to adopt the cookbook,
+    And I request to adopt the cookbook, but get no response from the cookbook owner,
+    I want the ability to have ownership transferred to me,
+    So that I can update the cookbook
+
+    As a supermarket user,
+    When I find a cookbook that has not been updated in some time,
+    And the cookbook is not up for adoption,
+    And no response comes from the owner when I or the Supermarket team attempt to contact them,
+    I want a process that is fair to the current cookbook owner, myself, and the larger Chef community,
+    To allow me to take ownership of the cookbook
+```
+
+## Specification
+
+### When the Cookbook is available for adoption
+
+When a cookbook owner puts the cookbook up for adoption, that means they no longer wish to be the owner and maintainer of the cookbook.  When the owner puts the cookbook up for adoption, anyone who visits the cookbook's page on Supermarket will see a blue button that says "Adopt me!"  When a user clicks this button, an email is sent to the cookbook's current owner.  Occasionally, however, the cookbook's current owner does not respond to the email requests for adoption.
+
+If a user has requested to adopt a cookbook and has not received a response from the cookbook's current owner within a week, they should reach out to the Supermarket team via email (currently nshamrell@chef.io and rkidd@chef.io).  The Supermarket team will then email the current owner directly.  If the Supermarket team does not receive a response from the current owner within an additional week, the Supermarket team will transfer ownership to the user who wishes to adopt the cookbok.
+
+### When the Cookbook is not available for adoption
+
+When a cookbook owner has NOT put a cookbook up for adoption, there is no concrete indication that they wish to transfer ownership of the cookbook.  However, occasionally cookbooks are abandoned and there is no current way to arbitrate transferring ownership to someone else.
+
+If a cookbook has not been updated in 3 months and a user is interested in taking ownership of the cookbook, they should reach out to the Supermarket team via email (currently nshamrell@chef.io and rkidd@chef.io).  The Supermarket team will then make every reasonable effort (including but not limited to email) to contact the cookbook's current owner.  If no response comes from the cookbook's current owner within two weeks of the Supermarket team attempting to contact them, the Supermarket team will transfer ownership to the user who wishes to take ownership of the cookbook.
```

Comment:
I've updated the pull request to indicate that the criteria/time period requirements for transferring ownership should be decided on by the community.